frontend/src/metadata/popups/SliceFinderPopup.svelte

```diff
@@ -244,16 +297,20 @@
 			</div>
 			<div class="generation" style="margin-bottom:0px;">
 				<h4 style="margin-bottom:0px;">Filter Predicates</h4>
-				<h4 style="margin-bottom:0px;">Slice Average Metrics</h4>
+				<h4 style="margin-bottom:0px;">
+					Slice Average Metric {$tab !== "comparison" ? "" : "difference"}
+				</h4>
 			</div>
 		{:else}
 			<div id="initial">
 				<span class="intial-text" style="font-weight: bold">
-					Click below to find slices with low performance!
+					Click below to find slices with {$tab !== "comparison"
```

Comment:
"Find slices with the lowest performance"
"Find slices with the largest difference"